source/_ecosystem/ios/notifications/basic.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: "Basic Notifications"
+description: "Basic notes about iOS notifications"
+date: 2016-10-25 15:00:00 -0700
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+The iOS notify platform accepts the standard `title`, `message` and `target` parameters.
```

Comment:
No screenshots on this page? This is like one of the first notifications pages people go too. It's fine to re-use an image